### ISSUE TITLE
What does "Remote save. An error has occurred, see your console for more informations." actually mean

### ISSUE THREAD

**Issue by jaredmorgs** [HubPress/hubpress.io#115]
I've been working with a colleague who is trying out HubPress, and he got the message in $Subject when trying to save his first blog post.

There are a few problems with this message:

* **An error has occurred** this needs to be more specific if possible. If HubPress is having trouble parsing the blog post because of incorrect AsciiDoc syntax, it should tell the user that.
* **See your console for more informations** doesn't tell you where this console is. I haven't been able to find it in the HubPress Admin console, which is where I would presume it resides.

If this error message relates to issues with the blog post content then the following error message would be better:

*Invalid AsciiDoc syntax detected. Check the blogpost for syntax errors an try again.*

If the error message relates to a HubPress back-end commit function failing because of credential issues or misconfiguration, the following error would be more informative:

*A configuration issue is preventing your blog from being changed. Review the README.adoc and check settings*

**Comment by alxndr** [comment]
Currently happening to me as well when trying to publish an edit or a new post. In the console I've found* this error:
```
{
  "message": "Maximum number of login attempts exceeded. Please try again later.",
  "documentation_url": "https://developer.github.com/v3"
}
```

---

\* In Chrome 40 on OS X 10.8.5, the developer tools can be shown via the menu bar "View" > "Developer" > "Developer Tools", or by pressing option-command-i. The developer tolos are a panel which has several tabs within it; the Console tab is where things are often logged and where you can execute JavaScript live. I found this error in the Network tab, which shows info about the requests that the browser sends out.

**Comment by anthonny** [comment]
:+1: @jaredmorgs I agree

@alxndr could you create a new issue for this problem and could you describe the steps to reproduce this error please ?

**Comment by rootasjey** [comment]
It happened to me as well when I was trying to save settings in the HubPress Admin console.
Maybe it is because of inactivity, i was logged in for 20 min approximatively.
A logout/login solve this issue for me.

**Comment by jaredmorgs** [comment]
Closing off because this issue appears to be fixed by #117
